### ISSUE TITLE
Lambda - cookies are not retrievable via SSR

### ISSUE THREAD

**Issue by acidjazz** [nuxt/framework#3803]
### Environment

AWS Lambda -> API Gateway -> ( CloudFront / S3 )

### Reproduction

I will update this w/ an example deployed setup as soon as I get time

### Describe the bug

I set a token cookie when the user logs in - when I deploy to Lambda that cookie is `undefined` via SSR, but found on the client-side, this results in the bug of the layout rendering a page twice.

I use `universal-cookie` to set and retrieve these cookies

setting it:

```ts
  async login (result: UserLogin): Promise<undefined|string> {
    this.loggedIn.value = true
    this.token.value = result.token
    Object.assign(this.$user, result.user)
    this.cookies.set('token', this.token.value, { path: '/', maxAge: 60*60*24*30 })
...
```

retrieving it:

```ts
  private getToken(): string {
    if (this.config.req) return useCookie(this.config.req, 'token')
    return this.cookies.get('token')
  }
```

### Additional context

This is only an issue with Lambda - ec2 and locally it works perffectly

**Comment by danielroe** [comment]
This should be resolved in nitropack v0.5.0 which you can try on the [edge channel](https://v3.nuxtjs.org/guide/going-further/edge-channel/), or in the next RC. 

Let me know if not with a reproduction and I'll reopen.

**Comment by acidjazz** [comment]
@danielroe apologies - this has been confirmed working for a while rc-4+ at least
